sycl/test-e2e/Matrix/SPVCooperativeMatrix/SG32/element_wise_all_sizes.cpp

```diff
@@ -5,9 +5,12 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
 
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
```

Comment:
was this failing on DG2  as it seems it was running on dg2?

Reply:
SG32 was never running on DG2, as it is not supported there for joint matrix. For this specific test it was guarded by using `matrix` feature (on DG2, instead of `matrix` feature, we were passing `matrix-xmx8`). Since I replace `matrix` with aspect, the test started to run and fail on DG2.